jsapp/js/theme/kobo/ActionIcon.ts

```diff
@@ -0,0 +1,38 @@
+import {rem, ActionIcon} from '@mantine/core';
+import classes from './ActionIcon.module.css';
+
+export const ActionIconThemeKobo = ActionIcon.extend({
+  classNames: classes,
+  vars: (theme, props) => {
+    return {
+      root: {
+        '--ai-size-sm': rem(28),
+        '--ai-size-md': rem(32),
+        '--ai-size-lg': rem(38),
+
+        ...(props.variant === 'filled' && {
+          '--ai-hover': theme.colors.blue[5],
+        }),
+        ...(props.variant === 'light' && {
+          '--ai-color': theme.colors.blue[5],
```

Comment:
I think that the blue color of the icon is lighter than our current icon buttons.

Reply:
> let's prefer CSS Modules file over styles prop, because that edits CSS class once, instead of adding `style="font-size: calc(0.875rem * var(--mantine-scale)); border-top: 1px solid rgb(237, 238, 242);"` to every cell individually (see DOM of storybook). Also, that's officially recommended.
> 
> Furthermore, would you agree to prefer vars prop over CSS Modules file, because that's a simple parameterization of the CSS Modules? Although performance is unknown and official docs lacks an opinion on using vars prop vs "hardcoded" values in CSS Modules.

Preferring vars over css is an option, let's talk in the meetup about our policy (options: prefer vars, prefer css, undefined).